src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -93,7 +94,8 @@ export default function TypeOfCarePage() {
     },
     uiSchema: {
       typeOfCareId: {
-        'ui:widget': 'radio',
+        'ui:title': pageTitle,
```

Comment:
Modified to use a custom radio widget since the radio buttons need to be 'grouped' in a fieldset for the desired heading to be read by VO.

Note, using the radio web component breaks existing unit test due to the need to read the shadow DOM.